maven-jxr/pom.xml

```diff
@@ -100,6 +100,22 @@ under the License.
         <groupId>org.eclipse.sisu</groupId>
         <artifactId>sisu-maven-plugin</artifactId>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>analyze</id>
+            <goals>
+              <goal>analyze-only</goal>
+            </goals>
+            <configuration>
+              <failOnWarning>true</failOnWarning>
+              <ignoredDependencies>org.slf4j:slf4j-simple:*</ignoredDependencies>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
```

Comment:
we can try to put it in root pom ...

Reply:
That doesn't buy very much since we still need to configure the ignoredDependencies on a project by project basis. That allows us to omit the goals and failOnWarning elements, but nothing else.